core/components/atoms/input/__stories__/index.story.jsx

```diff
@@ -24,7 +24,7 @@ export const all = () => {
 
   const error = false;
```

Comment:
please remove the unnecessary prop from the story which are by default `false` or does not have any effect on the Input like `onClear` callback

Reply:
Done